src/js/other-websites/inpage_menu.js

```diff
@@ -578,21 +765,113 @@ const buildContent = {
       const relayMenuDashboardLink = document.querySelector(
         ".fx-relay-menu-dashboard-link"
       );
-    
+
       const relayMenuDashboardLinkTooltip =
-        relayMenuDashboardLink.querySelector(".fx-relay-menu-dashboard-link-tooltip");
-    
-        relayMenuDashboardLinkTooltip.textContent =
+        relayMenuDashboardLink.querySelector(
+          ".fx-relay-menu-dashboard-link-tooltip"
+        );
+
+      relayMenuDashboardLinkTooltip.textContent =
         browser.i18n.getMessage("labelManage");
-    
+
       relayMenuDashboardLink.href = `${relaySiteOrigin}?utm_source=fx-relay-addon&utm_medium=input-menu&utm_content=manage-all-addresses`;
       relayMenuDashboardLink.target = "_blank";
-    
+
       relayMenuDashboardLink.addEventListener("click", async () => {
         sendInPageEvent("click", "input-menu-manage-all-aliases-btn");
         await iframeCloseRelayInPageMenu();
       });
-    }
+    },
+    search: {
+      inputFocus: async () => {
```

Comment:
nit (non-blocking): this function is doing more than just focusing on the search input - it's also updating the visible/total text. Maybe a better function name to match what it's doing?

Reply:
Renamed function to `initResultsCountAndFocusOnInput()`. Revised in d5e6221fe717faa6809d549a9e0b85259f67a063